disnake/enums.py

```diff
@@ -1344,6 +1361,11 @@ class EntitlementType(Enum):
     application_subscription = 8
 
 
+class PollLayoutType(Enum):
+    default = 1
+    """The default poll layout type."""
```

Comment:
This is already documented in `disnake.Event` page and in the messages API reference section too. Is there somewhere else this should be documented?

Reply:
I believe what @Victorsitou meant was moving this docstring entirely into `messages.rst`, since `.. autoclass::` and `:members:` unfortunately looks a bit off with these enums:
![image](https://github.com/user-attachments/assets/f1557e09-446a-4e85-8e99-e3ff5f5b6790)
compared to another one, for example:
![image](https://github.com/user-attachments/assets/02b2488e-7035-4882-b36c-8dfd2d707896)